proto/wg/cosmo/graphqlmetrics/v1/graphqlmetrics.proto

```diff
@@ -0,0 +1,48 @@
+// https://protobuf.dev/programming-guides/style/
+
+syntax = "proto3";
+
+package wg.cosmo.graphqlmetrics.v1;
+
+import "wg/cosmo/common/common.proto";
+
+message SchemaUsageInfo {
+  string OperationDocument = 1;
+  repeated TypeFieldUsageInfo TypeFieldMetrics = 2;
+  OperationInfo OperationInfo = 3;
+  RequestInfo RequestInfo = 4;
+  map<string,string> Attributes = 6;
+}
+
+message OperationInfo {
+  string OperationHash = 1;
+  string OperationName = 2;
+  string OperationType = 3;
```

Comment:
Make Type an Enum?

Reply:
Requires additional mapping but could be worth it from a protocol perspective.

Reply:
Done